src/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] I think the add_rental should take in the `person` and `date` as parameters and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. like this :point_down: 

  ```ruby
    def add_rental(person, date)
      Rental.new(date, self, person)
    end
  ```

Reply:
Fixed.